packages/block-library/src/image/image.js

```diff
@@ -504,25 +564,20 @@ export default function Image( {
 				} }
 				ref={ imageRef }
 				className={ borderProps.className }
-				style={ borderProps.style }
+				style={ {
+					width:
```

Comment:
Did we need a deprecation here?

It's causing some deprecation errors on existing blocks that do not have the styles set.

See https://github.com/WordPress/gutenberg/issues/52388#issuecomment-1631721808

Reply:
@ramonjd This is part of `edit`, not `save`, so it shouldn't affect block deprecations. In a follow-up https://github.com/WordPress/gutenberg/pull/52286, @draganescu added styles to `save`, but he did add a deprecation for them.

Reply:
Doh! My bad for overseeing it. Thanks for the quick response 🙇🏻